### DISCUSSION TITLE
22621.3007.63.4 will not install

### DISCUSSION THREAD

**Discussion by ruebennase56** [valinet/ExplorerPatcher#2905] [Q&A]
Hello to everyone,
I encountered a problem. Versions 63.3 and 63.4 won't install on my Windows 11 setup (23H2 22631.3155). And both Version are still flagged as a Trojan by Microsoft Defender (not a big thing Defender does that from time to time).

Has anyone the same problem and maybe a solution?

English translation of the requester: "This app can not be run on the PC. Contact the software publisher to obtain a suitable version."
![Unbenannt](https://github.com/valinet/ExplorerPatcher/assets/88430929/290d1a33-f946-4e14-b440-55fcf812f347)

**Comment by pyrates999** [comment]
False flag.  See here: https://github.com/valinet/ExplorerPatcher/issues/2873
